security/src/main/java/org/kaazing/gateway/security/auth/BaseStateDrivenLoginModule.java

```diff
@@ -141,5 +153,18 @@ public boolean logout() throws LoginException {
 
     protected abstract boolean doLogout() throws LoginException;
 
+    private LoginResult getLoginResultFromCallback() {
+        LoginResultCallback loginResultCallback = new LoginResultCallback();
+
+        try {
+            this.handler.handle(new Callback[]{loginResultCallback});
+        } catch (IOException|UnsupportedCallbackException e) {
+            LOGGER.debug("Encountered exception handling loginResultCallback", e);
```

Comment:
This is a nit, I would suggest the same capitalization for other exceptions in this PR.